scripts/utils/generate_strings.py

```diff
@@ -388,10 +218,6 @@ def serialize(string):
         help='YAML strings file to process')
     parser.add_argument('-o', '--output', metavar='DIR', type=str, action='store',
         help='Output directory for generated files')
-    parser.add_argument('-g', '--guides', metavar='DIR', type=str, action='store',
-        help='Parse JSON guides from DIR')
-    parser.add_argument('-t', '--tutorials', metavar='DIR', type=str, action='store',
-        help='Parse JSON tutorials from DIR')
     args = parser.parse_args()
 
     ## If no source was provided, find it relative to this script file.
```

Comment:
I can't comment on the actual string, but please fix the fallback to actually use rootpath (as explained in the l10n original issue)

Reply:
About this line though: comments should only have one `#`.